src/assets/scripts/client/aircraft/AircraftModel.js

```diff
@@ -168,6 +168,17 @@ export default class AircraftModel {
          */
         this.transponderCode = 1200;
 
+        /**
+         * Pilot's voice
+         *
+         * Initially generated and assined on instantiation by the `AircraftController`
+         *
+         * @for AircraftModel
+         * @property pilotVoice
+         * @type {string}
+         */
+        this.pilotVoice = randomize_pilot_voice();
```

Comment:
Just as a note, usually we would instantiate the property empty, then populate it in an `.init()` method, and reset it in a `.reset()` method. That way, it would be easily recycled and reused by the `ModelSourceFactory`. Sadly, we do not do this with the `AircraftModel` yet, which is a big missed opportunity that we can address sometime 😃 

Since we don't currently do so, this is perfectly good! 👍

Reply:
We can refactor when we address this issue later.